### PR TITLE
Added MixChill(music player) files

### LINKED ISSUE #358: Want to add a music player
### Is your feature request related to a problem? Please describe.

I want to add an music player that I had built to listen to my top 5 favorite songs.

### Describe the solution you'd like.

I had used HTML, CSS, and JavaScript to make this responsive website.

### Describe alternatives you've considered.

This includes play, pause, previous and next.

### Add any other context or screenshots about the feature request here.

Screenshot 2022-03-01 143937
Screenshot 2022-03-01 144021

### PR DESCRIPTION
**Pull request by Anchita15**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Yes, I have read..)

### Description

(MixChill is a music player that I have made in order comprise my top 5 favorite songs and this website have some amazing features like play, pause, auto, volume etc.
This website has been made using HTML, CSS and JavaScript.
This is responisve website.)

Fixes #358 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

Screenshot 2022-03-01 143937
Screenshot 2022-03-01 144021

### CODE REVIEW

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Anchita15 Everything looks fine except , it would be great if you add some gif or may be pictures in `readme.md` file .

And rename your project name `Mixchill` to `Micchill(Music player)` or something else so that anyone can understand what exactly inside .

**Comment by Anchita15** [comment]
> 

Okay I'll do the desired changes.

**Comment by Anchita15** [comment]
> @Anchita15 Everything looks fine except , it would be great if you add some gif or may be pictures in `readme.md` file .
> 
> And rename your project name `Mixchill` to `Micchill(Music player)` or something else so that anyone can understand what exactly inside .

Okay I'll do the desired changes.

**Comment by Kushal997-das** [comment]
@Anchita15 Are you going to do the changes ? Or else your PR will close .

**Comment by Anchita15** [comment]
> @Anchita15 Are you going to do the changes ? Or else your PR will close .

I'll do it by today.

File: Web Development/Basic/MixChill(Music_Player)/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- HTML
+- CSS
+- Javascript
+
+## 📷 Demo Screenshots <a name = "Screenshots"></a>
+
+<div align="center">
+   <img src="MixChill(Music_Player)/images/ss1.png"></a>
+   <img src="MixChill(Music_Player)/images/ss2.png"></a>
```
Comment (Kushal997-das) [inline Web Development/Basic/MixChill(Music_Player)/readme.md:19]: Both the links not working !

**Review by Kushal997-das — COMMENTED** [review]
@Anchita15 Links not working !

**Review by Kushal997-das — APPROVED** [review]
@Anchita15 Great 💯

**Pull request merged**